_posts/developers/pod-server/2019-01-01-00_overview.md

```diff
@@ -12,7 +12,12 @@ redirect_from:
 
 # Setting up Node Solid Server
 
-There is currently one publicly available Pod server available that you can run yourself: [Node Solid Server](https://github.com/solid/node-solid-server). Instructions on how to set it up and how to get help are available [at its website](https://github.com/solid/node-solid-server), and some documentation is also provided below.
+There are currently three publicly available Pod server implementations available that you can run yourself:
+* [Node Solid Server](https://github.com/solid/node-solid-server) (JavaScript)
+* [Community Solid Server](https://github.com/solid/community-server) (TypeScript)
+* [Nextcloud](https://github.com/pdsinterop/solid-nextcloud) / [PHP Solid Server](https://github.com/pdsinterop/php-solid-server) (PHP)
```

Comment:
Is this not called "solid-nextcloud"? Don't want to give the impression that Nexcloud supports Solid out of the box, I suppose?

```suggestion
* [solid-nextcloud](https://github.com/pdsinterop/solid-nextcloud) / [PHP Solid Server](https://github.com/pdsinterop/php-solid-server) (PHP)
```